app/invocation/child_invocation_card.tsx

```diff
@@ -52,14 +53,38 @@ export default class ChildInvocationCard extends React.Component<ChildInvocation
     }
   }
 
+  private simplifyCommandLine(commandLine: string): string {
+    // Remove the bazel command string
+    if (commandLine.startsWith("bazel ")) {
+      commandLine = commandLine.slice("bazel ".length);
+    }
+
+    // Strip all --remote_header flags
+    commandLine = commandLine.replace(/\s*--remote_header=[^\s]+/g, "");
+    // Strip the options we manually added
+    commandLine = commandLine.replace(" --config=buildbuddy_bes_backend", "");
+    commandLine = commandLine.replace(" --config=buildbuddy_bes_results_url", "");
+    commandLine = commandLine.replace(" --config=buildbuddy_remote_cache", "");
+    commandLine = commandLine.replace(/\s*--invocation_id=[^\s]+/g, "");
+
+    return commandLine;
```

Comment:
We have `EXPLICIT_COMMAND_LINE` that is intended as a way to override the bazel command displayed in the UI in cases where there is some extra arg fiddling going on (e.g. the CLI) - can we set that build metadata and use it directly instead of "undoing" the arg fiddling here?

Reply:
I realized that the redactor strips EXPLICIT_COMMAND_LINE from the build metadata, so that is actually an ineffective way to override the bazel command displayed in the UI (Here's a [PR](https://github.com/buildbuddy-io/buildbuddy/pull/7734) to remove the code from the CLI. Alternatively, we could add it back and risk there being places we are incorrectly redacting it, which was the [original impetus to remove it](https://github.com/buildbuddy-io/buildbuddy/pull/4525))

Reply:
> the [redactor strips EXPLICIT_COMMAND_LINE from the build metadata](https://github.com/buildbuddy-io/buildbuddy/blob/master/server/util/redact/redact.go#L196)

I think the linked code is redacting the CommandLine event, not the BuildMetadata event?

Reply:
after a closer look, that linked code is stripping any nested occurrences of `--build_metadata=EXPLICIT_COMMAND_LINE=` within the explicit command line (which shouldn't normally happen), but not the EXPLICIT_COMMAND_LINE build metadata is still preserved in the `BuildMetadata` event